### PR TITLE
add Sign Up, Log In, Log Out functionality

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add code for the Sign-Up, login, and Log Out functionality.

Some of the code has been copied from past projects, so you can quickly look.

Please review and give your feedback.

### CODE REVIEW

File: src/components/compose/Compose.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+
```
Comment (animeshgarai09) [inline src/components/compose/Compose.jsx:2]: Can you explain what this component does? seems something new

**Review by animeshgarai09 — APPROVED** [review]
Everything seems clean and pracise. good to merge

File: src/components/compose/Compose.jsx (hunk 1 above)
Comment (hsnice16) [reply src/components/compose/Compose.jsx:2]: Sure, 
As the number of providers in `src/index.js` file gets larger, the code readability starts to decrease, what I think. So, to tackle that I am using this component.

As the number of providers get larger, the file starts to become something like
```jsx
<ProviderOne>
     <ProviderTwo>
           <ProviderThree>
 ... so on
```

This component is to solve this issue.

**Review by partha8 — APPROVED** [review]
Pretty straightforward PR, readable and clean. Liked the part where you set a maxage for cookie. Good to merge.

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**